projects/components/src/multi-select/multi-select.component.scss

```diff
@@ -89,6 +89,7 @@
   min-width: 120px;
   display: flex;
   height: 100%;
+  max-height: 360px;
```

Comment:
I think this should be driven by a component input. 360px might be too less in some cases

Reply:
also, we should make it scroll

Reply:
It already scrolls.

360px is good for about 10 items. Do we want the consumer to be able to control the height via an input? Currently, it can be done with css override.

Reply:
no. I meant we should be able to do it via code.

Reply:
umm. don't want to block the pr though